### PR TITLE
Added translated readme file (brazilian portuguese)

### PR DESCRIPTION
**Pull request by Felipefpl**
Added a brazilian portuguese translation of the readme file which was missing.

### CODE REVIEW

**Comment by endrift** [comment]
cc @yuriks to review whenever you have time

**Comment by Felipefpl** [comment]
Is there any problem with my pull or all the others? it's been more than one month and no answer for them.

**Comment by endrift** [comment]
I was hoping to get @yuriks, who also know Brazilian Portuguese to give this a look over before it's merged. She had some comments on previous translations you'd contributed so I was hoping to get her to look at it when she had time.

**Comment by Felipefpl** [comment]
> I was hoping to get @yuriks, who also know Brazilian Portuguese to give this a look over before it's merged.

Is she brazilian or just some person who has some knowledge of brazilian portuguese?

**Comment by yuriks** [comment]
I'm Brazilian. Either way, the reason I've left this sitting is that I lost whatever motivation I had to review translations because of the churn in the UI translations. Months ago I had spent a significant amount of time reviewing older translation PRs and checking messages in the context of the UI to make sure that they made sense, as well as trying to figure out good replacements for terms which have no direct translation, and it was pretty frustrating to get another batch of updates that seemingly arbitrarily shuffled the wording of most of the UI again with no clear rationales behind it, and even re-introducing some of the context errors I'd pointed out when reviewing the initial translations. So I decided to skip on reviewing the previous PR and ended up not really having motivation to review this when it got submitted either, and ended up forgetting about it after putting it off.

The readme isn't super long so I'll just quickly cross-check this within the next few hours and ack if it looks good, sorry for the delays.

File: README_PT_BR.md [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+<h2>Copyright</h2>
+
+<p>O mGBA tem um Copyright de © 2013 – 2021 do Jeffrey Pfau. É distribuído sob a <a href="https://www.mozilla.org/MPL/2.0/">Licença Pública do Mozilla, versão 2.0</a>. Uma cópia da licença está disponível no arquivo distribuído da LICENÇA.</p>
+
+<p>O mGBA contém as seguintes bibliotecas de terceiros:</p>
+
+<ul>
+<li><a href="https://github.com/benhoyt/inih">inih</a>, a qual tem um copyright de © 2009 – 2020 do Ben Hoyt e é usada sob uma licença de 3 claúsulas do BSD.</li>
+<li><a href="https://code.google.com/archive/p/blip-buf">blip-buf</a>, a qual tem um copyright de © 2003 – 2009 do Shay Green e é usada sob uma Licença Pública Inferior do GNU.</li>
+<li><a href="http://www.7-zip.org/sdk.html">SDK do LZMA</a>, a qual é de domínio público.</li>
```
Comment (yuriks) [inline README_PT_BR.md:270]:
```suggestion
<li><a href="http://www.7-zip.org/sdk.html">SDK do LZMA</a>, o qual está sob o domínio público.</li>
```

File: README_PT_BR.md [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+<li><a href="https://github.com/aappleby/smhasher">MurmurHash3</a> implementação do Austin Appleby, a qual é de domínio público.</li>
```
Comment (yuriks) [inline README_PT_BR.md:271]:
```suggestion
<li>Implementação da <a href="https://github.com/aappleby/smhasher">MurmurHash3</a> por Austin Appleby, a qual está sob o domínio público.</li>
```

File: README_PT_BR.md [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+<li><a href="https://github.com/skandhurkat/Getopt-for-Visual-Studio/">getopt pro MSVC</a>, a qual é de domínio público.</li>
```
Comment (yuriks) [inline README_PT_BR.md:272]:
```suggestion
<li><a href="https://github.com/skandhurkat/Getopt-for-Visual-Studio/">getopt pro MSVC</a>, a qual está sob domínio público.</li>
```

File: README_PT_BR.md [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+<li><a href="https://www.sqlite.org">SQLite3</a>, a qual é de domínio público.</li>
```
Comment (yuriks) [inline README_PT_BR.md:273]:
```suggestion
<li><a href="https://www.sqlite.org">SQLite3</a>, a qual está sob o domínio público.</li>
```

File: README_PT_BR.md [hunk 5]
```diff
@@ -0,0 +1,276 @@
...
+</ul>
+
+<p>Se você é um publicador de jogos e desejar licenciar o mGBA pra uso comercial por favor envie um email pra <a href="mailto:licensing@mgba.io">licensing@mgba.io</a> pra mais informações.</p>
```
Comment (yuriks) [inline README_PT_BR.md:276]:
```suggestion
<p>Se você é um publicador de jogos e desejar licenciar o mGBA para uso comercial por favor envie um email para <a href="mailto:licensing@mgba.io">licensing@mgba.io</a> para mais informações.</p>
```

File: README_PT_BR.md [hunk 6]
```diff
@@ -0,0 +1,276 @@
...
+<li>Suporte pra relógio em tempo real até mesmo sem configuração.</li>
+<li>Suporte pra sensor solar pros jogos Boktai.</li>
+<li>Suporte pra Câmera do Game Boy e pra Impressora do Game Boy.</li>
+<li>Uma implementação embutida da BIOS e a habilidade de carregar arquivos externos da BIOS.</li>
+<li>Suporte pra turbo/avanço rápido segurando o Tab.</li>
+<li>Rebobinação segurando o Backquote.</li>
+<li>Frameskip, configurável até 10.</li>
+<li>Suporte pra screenshot.</li>
+<li>Suporte pra códigos de trapaça.</li>
+<li>9 slots pra savestates. Os savestates são também visíveis como screenshots.</li>
```
Comment (yuriks) [inline README_PT_BR.md:29]:
```suggestion
<li>9 slots para savestates. Os savestates são também visíveis como screenshots.</li>
```

File: README_PT_BR.md [hunk 7]
```diff
@@ -0,0 +1,276 @@
...
+<li>Nintendo Switch</li>
+<li>Wii</li>
+<li>PlayStation Vita</li>
+</ul>
+
+<p>É conhecido que outras plataformas tipo Unix tais como o OpenBSD funcionam bem mas não são testadas e não são totalmente suportadas.</p>
+
+<h3>Requerimentos do sistema</h3>
+
+<p>Os requerimentos são mínimos. Qualquer computador que consiga executar o Windows Vista ou mais novo deve ser capaz de lidar com a emulação. Suporte pra OpenGL 1.1 ou mais novo também é requerido, com o OpenGL 3.2 ou mais novo pros shaders e funções avançadas.</p>
```
Comment (yuriks) [inline README_PT_BR.md:102]:
```suggestion
<p>Os requerimentos são mínimos. Qualquer computador que consiga executar o Windows Vista ou mais novo deve ser capaz de lidar com a emulação. Suporte para OpenGL 1.1 ou mais novo também é requerido, com o OpenGL 3.2 ou mais novo para os shaders e funções avançadas.</p>
```

**Review by yuriks — CHANGES_REQUESTED** [review]
Sorry, this ended up taking longer than I expected so I couldn't finish it yesterday.

I left correction suggestions. I thought that the tone of some of the word choices in the original text tended to be too informal for documentation.

Also, why is the formatting in HTML? The original and all other translations are in Markdown format. This should probably be fixed unless there's a particular reason for it. (I don't really understand why you went through the trouble of converting it in the first place?)

Let me know if you disagree with any of the suggestions. I rarely write documents in Portuguese these days but I used references to try to make sure my grammar suggestions were correct at least.

**Comment by Felipefpl** [comment]
Well, i dont agree with those since the litteral translation doesnt make anyone from our land to misunderstand anything written, i'll close this and keep the translation stored to distribute someday.

**Pull request closed without merging**